### ISSUE TITLE
Fig Bugs not work and now disabled

### ISSUE THREAD

**Issue by sean-s419** [ChoGGi/SurvivingMars_CheatMods#65]
I get the message "Mod Fix Bugs has been disabled and is no longer available."

**Comment by ChoGGi** [comment]
Yeah, it got reported on paradox mods, I sent a msg. Hopefully the mod sees it before weekend...

**Comment by ChoGGi** [comment]
You can signout to bypass it till the mod is restored

https://www.reddit.com/r/SurvivingMars/comments/n2klhv/choggis_library_disabled_by_content_restrictions/
